src/main/java/org/project/springweb/model/CartItem.java

```diff
@@ -0,0 +1,34 @@
+package org.project.springweb.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "cart_items")
+@Getter
+@Setter
+public class CartItem {
```

Comment:
```suggestion
@SQLDelete(sql = "UPDATE cart_items SET is_deleted = true WHERE id=?")
@SQLRestriction("is_deleted=false")
public class CartItem {
```

Reply:
Discussed with mentor that this comment was written by mistake